src/app.coffee

```diff
@@ -66,6 +67,12 @@ class App
                     deviceName: deviceName
                     password: password
                 Devices.registerDevice options, next
+            (credentials, next) ->
+                password = credentials.password
+                config = file: true
+                filterSDK.setDesignDoc cozyUrl, deviceName, password, config, \
```

Comment:
I was not clear. I don't like to be a situation that requires to escape a newline. It breaks the flow of reading when you come back on this code later.

Often, it is possible to extract part of the computation in a variable. Maybe `setDesignDoc = filterSDK.setDesignDoc`